.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Issue Link
+
+<!--
+    Use a linking keyword and issue number. For example, this section could say:
+        Fixes #41
+    If there is no link, you may remove this section. However, larger PRs may be rejected if there is no corresponding issue.
+
```

Comment:
I'm wondering why we wouldn't mandate a corresponding issue? I feel like even if the issue was made just to appease a possible requirement of having a corresponding issue, I would think having that documentation would still be good practice. That's just my opinion though.

Reply:
I don't fully remember since I originally wrote this for Dependiject months ago. I'll loop in @wboyd600 to see if he remembers.